eternalcore-core/src/main/java/com/eternalcode/core/feature/home/HomePlaceholderSetup.java

```diff
@@ -50,8 +49,8 @@ void setUp(PlaceholderRegistry placeholderRegistry) {
     }
 
     private String homesLeft(Player targetPlayer) {
-        int homesLimit = this.homeManager.getHomesLimit(targetPlayer, this.pluginConfiguration.homes);
-        int amountOfHomes = this.homeManager.getAmountOfHomes(targetPlayer.getUniqueId());
+        int homesLimit = this.homeService.getHomeLimit(targetPlayer);
+        int amountOfHomes = this.homeService.getAmountOfHomes(targetPlayer.getUniqueId());
 
         return homesLeft(homesLimit, amountOfHomes);
```

Comment:
homesLeft może zwrócić ujemną wartość i robi to gdy gracz straci permisje do wyższego limitu home'ów

czyli:
gracz ma permisje na 3 home i je ustawia
np. ranga wygasa i ma teraz limit home na 1
homeleft wypluwa  1 - 3 czyli -2 i podaje do placeholdera

Reply:
![image](https://github.com/EternalCodeTeam/EternalCore/assets/65517973/a3ff6c9b-9577-40ea-92f1-4057e643d18d)
fixed